topics/webapp/index.md

```diff
@@ -0,0 +1,9 @@
+---
+aliases: web-application
+display_name: Webapp
+short_description: A web application is any computer program that performs a specific function by using a web browser as its client.
+topic: webapp
+related: angular, perl, xml, html
+wikipedia_url: https://en.wikipedia.org/wiki/Web_application
+---
+Web appliaction is any program that is accessed over a network connection using HTTP, rather than existing on user's device. Applications like Flash games, online calculators, calendars, gmail, facebook, etc.
```

Comment:
`Web appliaction` => `A web application`

What about further describing the difference between a web app and other kinds of apps? I'm thinking the first sentence could become:

> A web application is any program that is accessed over a network connection using HTTP, usually via a [web browser](https://github.com/topics/browser). Web apps do not have to be installed on a user's device; they are run from a remote server. This differs from mobile apps, which are installed on the user's mobile device, and desktop applications, which are installed on the user's computer.

For the last sentence, how about:

> Common examples of web apps include Flash games, online calculators, calendars, Gmail, and Facebook.